highfidelity/hifi-content
# Sync web overlays in stage manager

**Test Plan**
1. In a domain where you have rez rights, run https://hifi-content.s3-us-west-1.amazonaws.com/liv/Prototypes/StageManager/stageManager.js

2. Open the stage manager app and paste in a URL , then click SET

3. Spawn a web overlay using the tablet controls and confirm that the overlay goes to your specified URL

4. Remove the web overlay and confirm that the overlay goes away

Should test for multiple people in the domain. The overlay spawns around {x: 98.5507, y: 6.56, z: 23.7452}
